tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -76,7 +76,7 @@
           ssh \
             -i {{ edpm_privatekey_path | default("/tmp/ansible_private_key") }} \
             {{ edpm_user }}@"${computes[$name]}" \
-            "grep -qF $uuid /var/lib/nova/compute_id || (echo $uuid | sudo tee /var/lib/nova/compute_id && sudo chown 42436:42436 /var/lib/nova/compute_id && sudo chcon -t container_file_t /var/lib/nova/compute_id)"
```

Comment:
I remove the grep part to simplify things. If the file is already there with the correct uuid then writing it again should not cause any change.

Reply:
I removed the container_t labeling of the file as in a greenfield deployment the file gets labeled with system_u:object_r:nova_var_lib_t:s0 instead.
```
[root@edpm-compute-0 ~]# stat /var/lib/nova/compute_id 
  File: /var/lib/nova/compute_id
  Size: 37        	Blocks: 8          IO Block: 4096   regular file
Device: fc01h/64513d	Inode: 20971652    Links: 1
Access: (0400/-r--------)  Uid: (42436/    nova)   Gid: (42436/    nova)
Context: system_u:object_r:nova_var_lib_t:s0
Access: 2024-03-19 15:59:03.534671483 +0000
Modify: 2024-03-19 15:58:57.582465452 +0000
Change: 2024-03-19 15:58:57.721470248 +0000
 Birth: 2024-03-19 15:58:57.582465452 +0000
```
Also added an extra stat of the file first to see the file attributes if the file is already there.